### PR TITLE
color atom created

### PR DESCRIPTION
**Pull request by harishkumar-srijan**
color atom created

### CODE REVIEW

File: stories/Atom/Global/Color/color.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+<ul>
+<li>colors are showed in the Primary and Accents colors option, this variables are used in component.</li>
+<li>colors variables are saved in <code>assets/scss/variables</code> folder file in SCSS file format. and its managable by this file all over the site.</li>
+</ul>
+
+<div className="subheading">Primary Colors</div>
+
+Primary colours are the base colours of the UI, the ones that are used most frequently in the interface. it is used for brand elements and non-interactive elements. Interactive colours are used for primary links and action items that can be clicked or touched, to emphasize important actions and highlight information in your application.
+
+<div className="subheading">How to Use </div>
```
Comment (surbhi90) [inline stories/Atom/Global/Color/color.stories.mdx:95]: @harishkumar-srijan can we remove this section from here and add its content in the usage

File: stories/Atom/Global/Color/color.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+  .light-green {background-color: #B8ECB6;}
+  .green {background-color: #6DE354;}
+  .dark-green {background-color: #59BA47;}
+  .light-azure {background-color: #A2DAF3;}
+  .azure {background-color: #60D4F2;}
+  .dark-azure {background-color: #00C1FF;}
+
+`}</style>
+
+# Color Component
```
Comment (srijan-mehul-gada) [inline stories/Atom/Global/Color/color.stories.mdx:83]: Please use Grammarly to correct grammatical errors

**Review by srijan-mehul-gada — CHANGES_REQUESTED** [review]
Please review documentation again for grammatical error and also make sure it is having logical flow

File: stories/Atom/Global/Color/color.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+Color component represent the all colors we are used into the website. we used all the colors as variables.
```
Comment (srijan-mehul-gada) [inline stories/Atom/Global/Color/color.stories.mdx:84]: please use the below text:

This component represents various colors that can be used in the website as per Brand guidelines. All these colors are defined as SCSS variables globally and accordingly used in all the components. The variables can be found in assets/scss/_variables.scss file. 

Colors are divided into two options - Primary and Accents colors.

**Review by srijan-mehul-gada — COMMENTED** [review]
update documentation text as mentioned in the comment

File: stories/Atom/Global/Color/color.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+  .dark-green {background-color: #59BA47;}
+  .light-azure {background-color: #A2DAF3;}
+  .azure {background-color: #60D4F2;}
+  .dark-azure {background-color: #00C1FF;}
+
+`}</style>
+
+# Color Component
+This component represents various colors that can be used in the website as per Brand guidelines. All these colors are defined as SCSS variables globally and accordingly used in all the components. The variables can be found in assets/scss/_variables.scss file.
+
```
Comment (surbhi90) [inline stories/Atom/Global/Color/color.stories.mdx:85]: @harishkumar-srijan Variables defined in scss point mentioned 2-3 times so just keep it in usage section

File: stories/Atom/Global/Color/color.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+
+<ul>
+<li>Colors are divided into two options - Primary and Accents colors.</li>
+<li>Colors variables are saved in <code>assets/scss/variables</code> folder file in SCSS file format. and its managable by this file all over the site.</li>
+</ul>
+
+<div className="subheading">Primary Colors</div>
+
+Primary colours are the base colours of the UI, the ones that are used most frequently in the interface. it is used for brand elements and non-interactive elements. Interactive colours are used for primary links and action items that can be clicked or touched, to emphasize important actions and highlight information in your application.
```
Comment (surbhi90) [inline stories/Atom/Global/Color/color.stories.mdx:94]: @harishkumar-srijan multiple colors spellings are used

File: stories/Atom/Global/Color/color.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+  {(args, { globals: { locale } }) => {
+    const caption = getCaptionForLocale(locale);
+    return <Color Primarytext={caption.PT}   PrimaryColor={caption.PC}  AccentColor={caption.AC} Balancepalette={caption.BC} ></Color>;
+  }}
+</Story>
+
+<div className="subheading">Usage</div>
+<ul>
+<li>We can use the color by taking the color code from the Documentation, like <code>#B5D5F5</code>, bbelow colors are defined with color code name. </li>
+<li>We can use by through variables that are is define in <code>assests/scss/variables.scss</code></li>
```
Comment (surbhi90) [inline stories/Atom/Global/Color/color.stories.mdx:275]: @harishkumar-srijan correct below spelling

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

**Review by surbhi90 — APPROVED** [review]
Looks good to me

**Pull request merged**